### PR TITLE
casmuser-2677: Add Ansible roles changes for NMN bonding

### PR DESCRIPTION
**Pull request by keopp-hpe**
#### Summary and Scope
This PR updates the UAN_Ansible_Roles.md file to include the new variables supporting bonded NMN and the new way CAN slaves are defined.  It also clarifies some of the wording around CAN and bonded NMN in Configure_Interfaces_on_UANs.md.

### CODE REVIEW

File: docs/portal/developer-portal/operations/Configure_Interfaces_on_UANs.md [hunk 1]
```diff
@@ -11,11 +11,15 @@ In the command examples of this procedure, `PRODUCT_VERSION` refers to the curre
...
+By default, a direct connection to the sites user network is assumed and the Admin must define the interface(s) and default route using the `customer_uan_interfaces` and `customer_uan_routes` structures. If `uan_can_setup` is a true value, user access will be over CAN or CHN depending on what the system default route is set to in SLS.
+
+* When CAN is set elected as the system default route in SLS, the bonded CAN interfaces are defined by `uan_can_bond_slaves` \(see [UAN Ansible Roles](UAN_Ansible_Roles.md)\) and the default route is set to the bonded CAN interface `can0`.
```
Comment (darrenn-jackson) [inline docs/portal/developer-portal/operations/Configure_Interfaces_on_UANs.md:16]:
```suggestion
* When CAN is set as the system default route in SLS, the bonded CAN interfaces are defined by `uan_can_bond_slaves` \(see [UAN Ansible Roles](UAN_Ansible_Roles.md)\) and the default route is set to the bonded CAN interface `can0`.
```

File: docs/portal/developer-portal/operations/UAN_Ansible_Roles.md [hunk 2]
```diff
@@ -222,6 +222,44 @@ None.
...
+the node PXE boots.
+
+**IMPORTANT!** The first interface in the list must be the `nmn0` interface which is configured during the
+initial image boot, typically `ens10f0`.  This is required because the MAC address of the `nmn0` interface
+is the MAC associated with the IP address of the UAN.  The bonded `nmnb0` interface and the bond slaves
+will assume this MAC and the IP address of `nmn0` to preserve connectivity.
+
+The second interface is typically the first port of a different 25Gb NIC for resiliency.
+
+The default values of `uan_nmn_bond_slaves are shown here.  They may need to be changed to match the actual
```
Comment (darrenn-jackson) [inline docs/portal/developer-portal/operations/UAN_Ansible_Roles.md:255]:
```suggestion
The default values of `uan_nmn_bond_slaves` are shown here.  They may need to be changed to match the actual

```

File: docs/portal/developer-portal/operations/UAN_Ansible_Roles.md [hunk 3]
```diff
@@ -238,6 +276,27 @@ The default value of `uan_can_setup` is `no`.
...
+`uan_can_bond_slaves` is a list of the interfaces to use as the bond slave pair when `uan_can_setup` is true
+and the Customer Access Network (CAN) is configured on the system.
+
+The interface names must be in a format which doesn't change between reboots of the node, such as `ens10f1`
+which is the second port of the NIC in slot 10.  
+
+**NOTE:** `ens10f1` is typically the second port of the OCP 25Gb card and is used as one of the bond
+slaves in the CAN `bond0` interface.
+
+The second interface is typically the second port of a different 25Gb NIC for resiliency.
```
Comment (darrenn-jackson) [inline docs/portal/developer-portal/operations/UAN_Ansible_Roles.md:290]: So if we can bond the 1st 25Gb NIC of the first OCP card with the 1st 25Gb NIC of the second OCP card for NMN, and we can bond the 2nd 25Gb NIC of the first OCP card with the 2nd 25Gb NIC of the second OCP card for CAN, then theoretically, a customer could have both bonding configs at the same time?

**Review by darrenn-jackson — CHANGES_REQUESTED** [review]
a few minor formatting fixes and one question for clarification.

File: docs/portal/developer-portal/operations/UAN_Ansible_Roles.md (hunk 3 above)
Comment (keopp-hpe) [reply docs/portal/developer-portal/operations/UAN_Ansible_Roles.md:290]: True.  This is what I did on surtur.  Bonded NMN on the first ports of two cards and bonded CAN on the second ports of those two cards.

File: docs/portal/developer-portal/operations/UAN_Ansible_Roles.md (hunk 2 above)
Comment (keopp-hpe) [reply docs/portal/developer-portal/operations/UAN_Ansible_Roles.md]: Addressed in latest commit.

File: docs/portal/developer-portal/operations/Configure_Interfaces_on_UANs.md (hunk 1 above)
Comment (keopp-hpe) [reply docs/portal/developer-portal/operations/Configure_Interfaces_on_UANs.md]: Addressed in latest commit.

File: docs/portal/developer-portal/operations/Configure_Interfaces_on_UANs.md [hunk 4]
```diff
@@ -11,11 +11,15 @@ In the command examples of this procedure, `PRODUCT_VERSION` refers to the curre
...
 
-By default, a direct connection to the sites user network is assumed and the Admin must define the interface(s) and default route using the `customer_uan_interfaces` and `customer_uan_routes` structures. If `uan_can_setup` is a true value, user access will be over CAN or CHN depending on what the system default route is set to in SLS.  When CAN is selected as the system default route in SLS, the CAN interfaces are defined by `uan_can_bond_slaves` \(see [UAN Ansible Roles](UAN_Ansible_Roles.md)\) and the default route is set to the bonded CAN interface `can0`. When CHN is selected as the system default route in SLS, the CHN IP is added to `hsn0` and the default route is set to the CHN. The Admin may override the CAN/CHN default route by setting `uan_customer_default_route` to true and defining the default route in `customer_uan_routes`.
+By default, a direct connection to the sites user network is assumed and the Admin must define the interface(s) and default route using the `customer_uan_interfaces` and `customer_uan_routes` structures. If `uan_can_setup` is a true value, user access will be over CAN or CHN depending on what the system default route is set to in SLS.
```
[the remark this answers is not in the document]
Comment (keopp-hpe) [reply docs/portal/developer-portal/operations/Configure_Interfaces_on_UANs.md]: Addressed in latest commit.

**Review by darrenn-jackson — APPROVED** [review]

**Review by alanm-hpe — APPROVED** [review]

**Pull request merged**